go/vt/vtgate/planbuilder/testdata/vschemas/schema.json

```diff
@@ -476,18 +480,11 @@
         "seq": {
           "type": "sequence"
         },
-        "unsharded_ref": {
-          "type": "reference"
```

Comment:
While having `type` set to `reference` is valid, it doesn't play a meaningful role in any of the planbuilder test cases. I think it's clearer to just have these be basic tables.